.github/workflows/github-release.yml

```diff
@@ -38,6 +38,18 @@ concurrency:
 run-name: ${{ inputs.node_tag || inputs.signer_tag }}
 
 jobs:
+  andon-cord:
+    if: |
+      inputs.node_tag != '' ||
+      inputs.signer_tag != ''
+    name: Andon Cord
+    runs-on: ubuntu-latest
+    environment: "Build Release"
+    steps:
+      - name: Check Approval
+        id: check
+        run: |
+          return true
```

Comment:
I'm not quite following what this is doing... can you explain in a comment?

Reply:
e2c923e94b305dc11f1b96dd36ffc2bababa6b38

Reply:
this also needed to be `exit`, not `return` since it wasn't a function. updated

Reply:
f9a31bd2011dcf85aeea69c438e6b4f51827e56a